services/app-graphql/src/schema.graphql

```diff
@@ -701,7 +705,7 @@ type CreateRateQuestionPayload {
 #    question: Question!
 #}
 
-input CreateContractQuestionResponseInput {
+input CreateQuestionResponseInput {
```

Comment:
Can you clarifying this is a generic type it works for both rate and question responses